RELEASE_NOTE.md

```diff
@@ -1,5 +1,21 @@
 # OpenPAI Release Note
 
+## July 2021 (version 1.8.0)
+
+- Marketplace related update
+  - Please see [Marketplace](https://github.com/microsoft/openpaimarketplace/releases/tag/v1.8.0) for more details
+
+- Alert manager
+  - Send alert to user when job failed #5337
```

Comment:
Send alert to users when job status changed #5337

Reply:
Done.